docs/v3.x/admin-panel/customization.md

```diff
@@ -154,21 +132,19 @@ export const SHOW_TUTORIALS = false;
 export const SETTINGS_BASE_URL = '/settings';
 ```
 
-### Changing the port
+### Changing the host and port
 
-By default, the front-development server runs on the `8000` port. However, you can change this setting by updating the following configuration:
+By default, the front-development server runs on port `localhost:8000`. However, you can change this setting by updating the following configuration:
 
 **Path —** `./config/server.js`.
 
 ```js
-module.exports = {
-  host: 'localhost',
-  port: 1337,
-  cron: {
-    enabled: false,
-  },
+module.exports = ({ env }) => ({
+  host: env('HOST', '0.0.0.0'),
+  port: env.int('PORT', 1337),
   admin: {
-    port: 3000,
+    host: 'my-host', // only used along with strapi develop --watch-admin
```

Comment:
need clarification as to what `my-host` is

Reply:
I agree and can you give me an example?
Because when you are using `--watch-admin` you are on your `localhost` nop?

Reply:
And maybe change that to make it more understandable

```diff
- // only used along with strapi develop --watch-admin
+ // only used along with `strapi develop --watch-admin` command
```

Reply:
I just rewritted what was already written : https://github.com/strapi/strapi/pull/6778/files/6cdf5e4e8a737dc9d2f9eebcad1b2d29dff22846#diff-348c3349a2ba00e769a06299eec0d20fL44

I was surprised to see that option too but it does exist. You can change the port and also change the host to use something else than localhost. I don't know the use case but the feature's here :p I guess it may be useful if you want to deal with subdomains.